compiler/src/lsp/output.re

```diff
@@ -0,0 +1,55 @@
+type lsp_error = {
+  file: string,
+  line: int,
+  startchar: int,
+  endline: int,
+  endchar: int,
+  lsp_message: string,
+};
+
+let lsp_error_to_yojson = (e: lsp_error): Yojson.t =>
```

Comment:
Sort of the same deal here, could we just derive it?

Reply:
That's done.  I'm not 100% confident with yojson stuff, not sure if I have to go from safe to basic to print it?